docs/psoc6/quickref.rst

```diff
@@ -236,3 +236,77 @@ See :ref:`machine.RTC <machine.RTC>` ::
     Setting a random week day in 'wday' field is not valid. The underlying library implements the logic to always
     calculate the right weekday based on the year, date and month passed. However, datetime() will not raise an error 
     for this, but rather re-write the field with last calculated actual value.
+
+Network Module
+--------------
+
+The :mod:`network` module
+
+See :ref:`network.WLAN <network.WLAN>`
+
+The network module is used to configure the WiFi connection.The WiFi interface for the station mode is only configured for
+this port.Create WLAN interface object using ::
+
+    import network
+    wlan = network.WLAN(network.STA_IF) # create station interface
+
+Scan for the available wireless networks using 
+
+::
+
+   wlan.scan()             
+    
+Scan function returns a list of tuple information about access points
+(ssid, bssid, channel, RSSI, security, hidden) .There are 7 levels of security.
+
```

Comment:
Suggestion: 
This part could be highlighted to draw attention. 
![image](https://user-images.githubusercontent.com/105282041/229752959-ab3fa6d4-cb81-405c-b77b-a512e1c23618.png)

Reply:
Thanks, @boramonideep! Done